presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderWindows.java

```diff
@@ -131,10 +131,11 @@
     public void testNonMergeableABAReordersToAABAllOptimizers()
     {
         @Language("SQL") String sql = "select " +
-                "sum(quantity) over(PARTITION BY suppkey ORDER BY orderkey ASC NULLS LAST) sum_quantity_A, " +
-                "avg(discount) over(PARTITION BY partkey ORDER BY receiptdate ASC NULLS LAST) avg_discount_B, " +
-                "min(tax) over(PARTITION BY suppkey ORDER BY shipdate ASC NULLS LAST) min_tax_A " +
-                "from lineitem";
+                "sum(quantity) over (w ORDER BY orderkey ASC NULLS LAST) sum_quantity_A, " +
```

Comment:
I think for these two basic tests, we should test both inline and referenced windows to know that both cases work.